lib/rubocop/cop/lint/redundant_type_conversion.rb

```diff
@@ -161,6 +168,11 @@ class RedundantTypeConversion < Base
           }
         PATTERN
 
+        # @!method exception_false?(node)
+        def_node_matcher :exception_false?, <<~PATTERN
```

Comment:
It may be redundant, but for your reference :-)

```suggestion
        def_node_matcher :exception_false_keyword_argument?, <<~PATTERN
```

Reply:
Updated.